### PR TITLE
Remove OMERO 4.0 versions from FLIMfit downloads page.

### PR DESCRIPTION
**Pull request by imunro**
Remove OMERO 4.0 versions from FLIMfit downloads page.

### CODE REVIEW

File: flimfit_downloads.html [hunk 1]
```diff
@@ -54,26 +54,31 @@
...
+                    <strong><a href="#flimfit_50">FLIMfit</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#mcr">MCR</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#code">Code</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#previous">Previous versions</a>
                     </strong></p>
                 <ul>
                     <li>
+                        <p>Please note that this version of FLIMfit is compatible
+                        with OMERO 5.0 servers. If you require compatibilty with
+                        an OMERO 4.0 server, earlier versions are available from
```
Comment (hflynn) [inline flimfit_downloads.html]: Should be an OMERO 4.4 server

**Comment by hflynn** [comment]
One minor issue but otherwise the new text reads fine.

**Comment by sbesson** [comment]
@imunro: could you also remove lines 42 and 43 from `flimfitgen.py` as part of this PR? See http://ci.openmicroscopy.org/job/FLIMfit-release-downloads/57/console

**Comment by sbesson** [comment]
Thanks. Last commit fixed http://ci.openmicroscopy.org/view/Consortium/job/FLIMfit-release-downloads/58/console

**Comment by sbesson** [comment]
@hflynn want to review https://downloads.openmicroscopy.org/flimfit/4.7.0/

**Comment by hflynn** [comment]
Slightly confused as to why we have a Windows installer for MCR when the text says that the Windows FLIMfit version installs it automatically. But otherwise :+1:

**Comment by imunro** [comment]
Fair point! Just in case ....

File: flimfit_downloads.html [hunk 2]
```diff
@@ -54,26 +54,31 @@
...
-                            4.4</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
+                    <strong><a href="#flimfit_50">FLIMfit</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#mcr">MCR</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#code">Code</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a
                             href="#previous">Previous versions</a>
                     </strong></p>
                 <ul>
                     <li>
+                        <p>Please note that this version of FLIMfit is compatible
+                        with OMERO 5.0 servers. If you require compatibilty with
```
Comment (sbesson) [inline flimfit_downloads.html]: `compatibility`

Comment (hflynn) [inline flimfit_downloads.html]: Sorry, i missed that!

**Comment by sbesson** [comment]
Merging for 4.7.0

**Comment by sbesson** [comment]
--rebased-to #122

**Pull request merged**